src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml

```diff
@@ -54,44 +57,71 @@ limitations under the License.
-    <attribute side="server" code="0x0003" define="COMMISSIONED_FABRICS" type="INT8U" writable="false" optional="false">CommissionedFabrics</attribute>
+    <attribute side="server" code="0x0000" define="NOCs" type="ARRAY" entryType="NOCStruct" length="320" writable="false" optional="false">
+      <description>NOCs</description>
+      <access op="read" privilege="administer"/>
+      <access modifier="fabric-scoped"/>
+    </attribute>
+    <attribute side="server" code="0x0001" define="FABRICS" type="ARRAY" entryType="FabricDescriptor" length="320" writable="false" optional="false">
+      <description>Fabrics</description>
+      <access op="read" privilege="view"/>
+      <access modifier="fabric-scoped"/>
+    </attribute>
+    <attribute side="server" code="0x0002" define="SUPPORTED_FABRICS" type="INT8U" writable="false" optional="false">
+      <description>SupportedFabrics</description>
+      <access op="read" privilege="view"/>
+    </attribute>
+    <attribute side="server" code="0x0003" define="COMMISSIONED_FABRICS" type="INT8U" writable="false" optional="false">
+      <description>CommissionedFabrics</description>
+      <access op="read" privilege="view"/>
+    </attribute>
     <!-- 400 = 400 bytes for root cert -->
-    <attribute side="server" code="0x0004" define="TRUSTED_ROOTS" type="ARRAY" entryType="OCTET_STRING" length="400" writable="false" optional="false">TrustedRootCertificates</attribute>
-    <attribute side="server" code="0x0005" define="CURRENT_FABRIC_INDEX" type="fabric_idx" writable="false" optional="false">CurrentFabricIndex</attribute>
+    <attribute side="server" code="0x0004" define="TRUSTED_ROOTS" type="ARRAY" entryType="OCTET_STRING" length="400" writable="false" optional="false">
+      <description>TrustedRootCertificates</description>
+      <access op="read" privilege="view"/>
+    </attribute>
+    <attribute side="server" code="0x0005" define="CURRENT_FABRIC_INDEX" type="fabric_idx" writable="false" optional="false">
+      <description>CurrentFabricIndex</description>
+      <access op="read" privilege="view"/>
+    </attribute>
 
     <command source="client" code="0x00" name="AttestationRequest" optional="false">
       <description>Sender is requesting attestation information from the receiver.</description>
       <arg name="AttestationNonce" type="OCTET_STRING"/>
+      <access op="invoke" privilege="administer"/>
     </command>
 
     <command source="server" code="0x01" name="AttestationResponse" optional="false">
       <description>An attestation information confirmation from the server.</description>
       <arg name="AttestationElements" type="OCTET_STRING"/>
       <arg name="Signature" type="OCTET_STRING"/>
+      <access op="invoke" privilege="administer"/>
     </command>
 
     <command source="client" code="0x02" name="CertificateChainRequest" optional="false">
```

Comment:
Add `response="CertificateChainResponse"`

Reply:
Did the comment refer to the already existing response?

```
    <command source="server" code="0x03" name="CertificateChainResponse" optional="false">
      <description>A device attestation certificate (DAC) or product attestation intermediate (PAI) certificate from the server.</description>
      <arg name="Certificate" type="OCTET_STRING"/>
    </command>
```